aiida_pseudo/cli/install.py

```diff
@@ -231,6 +231,23 @@ def cmd_install_pseudo_dojo(version, functional, relativistic, protocol, pseudo_
                 msg = f'md5 of pseudo for element {element} does not match that of the metadata {md5}'
                 echo.echo_critical(msg)
 
+        adjusted_cutoff_elements = []
+        for stringency, str_cutoffs in cutoffs.items():
+            max_cutoff_wfc = max([str_cutoffs[element]['cutoff_wfc'] for element in str_cutoffs])
+            max_cutoff_rho = max([str_cutoffs[element]['cutoff_rho'] for element in str_cutoffs])
+            for element, cutoff in str_cutoffs.items():
+                if cutoff['cutoff_wfc'] <= 0:
+                    cutoffs[stringency][element]['cutoff_wfc'] = max_cutoff_wfc * 2.0
```

Comment:
If we are already taking the maximum of the stringency set, should we really even double it? Can't find your comment anymore about what the PD team recommended about cutoffs for lanthanides. I have the feeling this is a bit excessive. If we give a warning that they are not officially tested, maybe just using the max itself should be fine?

Reply:
They recommended 20 Ha as a rule of thumb, but they also mentioned that the density (PAW double grid) cutoff may need to be larger than what is generally required. As the elements that are missing are only the Lanthanides, it could make sense to do something "too strict" to be safe.

Reply:
ok, but what is the max cutoff in the strict set? Is that more or less than 20 Ha. Because it if iis more, and we times two that, it will be _a lot_ more than 20 Ha, so maybe a bit excessive.

Reply:
I think the max in the strict set is 25, so I agree that 2.0 would be a bit extreme. Perhaps a lower multiplier would be better.